astropy/modeling/tests/test_blackbody.py

```diff
@@ -34,8 +34,8 @@ def test_evaluate(self, temperature):
         b = BlackBody1D(temperature=temperature,
                         bolometric_flux=bolometric_flux)
 
-        assert_quantity_allclose(b(1.4 * u.micron), 4734464.498937388 * u.Jy)
-        assert_quantity_allclose(b(214.13747 * u.THz), 4734464.498937388 * u.Jy)
+        assert_quantity_allclose(b(1.4 * u.micron), 4734463.93280426 * u.Jy)
```

Comment:
Here, I agree with the change (i.e., do not think we should have a larger `atol`), since this is presumably fixed for all eternity now!